app/models/wizard/steps/base.rb

```diff
@@ -18,6 +19,16 @@ def persisted?
         false
       end
 
+      protected
+
+      def next_step_path(*)
+        raise('Must implement this method')
```

Comment:
Ruby has a specific thing for non-compliance with interfaces: https://ruby-doc.org/core-2.5.0/NotImplementedError.html

Reply:
```suggestion
        raise NotImplementedError
```